composer.json

```diff
@@ -22,7 +22,7 @@
         "symfony/property-access": "~2.3",
         "twig/twig": "*",
         "knplabs/knp-markdown-bundle": "~1.3.2",
-        "egeloen/ckeditor-bundle": "~2.2",
+        "egeloen/ckeditor-bundle": "~3.0",
```

Comment:
> if a day for example we got a 2.8 version, it's possible to see `~2.2|~3.0|~4.2|~5.1`.

Yes, it's possible. But as @JHGitty said, older versions support should be dropped on next major release.